perun-base/src/main/java/cz/metacentrum/perun/core/api/exceptions/FacilityContactExistsException.java

```diff
@@ -1,46 +0,0 @@
-package cz.metacentrum.perun.core.api.exceptions;
-
-import cz.metacentrum.perun.core.api.Facility;
-
-/**
- * Checked version of FacilityContactExistsException
- *
- * @author Michal Stava <stavamichal@gmail.com>
- */
-public class FacilityContactExistsException extends EntityExistsException {
```

Comment:
While this is not used, I think it should be used in impl layer of ``FacilitiesManager``, in ``addFacilityContact()`` method. We do this for all entities, instead of throwing ``InternalErrorException``. Please check, if we have some constraint on table, then we could catch ``ConstraintViolationException`` (I'm not sure about the name) and wrap it as ``FacilityContactExistsException``.